File: Week-02/1. Primitive/Solutions/PrateekRai (CDE-1810256)/Solution-04.java
/*package whatever //do not write package name here */

import java.util.*;
import java.lang.*;
import java.io.*;

class GFG {
	public static void main (String[] args) {
		
		Scanner sc = new Scanner(System.in);
		
		    int t =sc.nextInt();
		    int i=0;
		    int k=0;
		    for(i=0;i<t;i++)
		    {
		        int n = sc.nextInt();
		        short[] a = new short[n]; 
		        
		        for(k=0;k<n;k++)
		        {
		            a[k] = sc.nextShort();
		        }
		        int max=a[0];
		        int min=a[0];
		        for(k=0;k<n;k++)
		        {
		            if(max<a[k])
		            {
		                max = a[k];
		            }
		            else if(min>a[k])
		            {
		                min = a[k];
		            }
		        }
		        System.out.println(max+" "+min);

		    }
		
		
	}
}

File: Week-02/1. Primitive/Solutions/Mukesh Kumar (Gcs-1840095)/Solution-04.cpp
#include<iostream>
using namespace std;
int main()
{
	int t,N;
	cin>>t;
	for(int k=1;k<=t;k++)
	{
	    cin>>N;
	    int a[N];
	    for(int i=0;i<N;i++)
	    cin>>a[i];
	    int max=a[0];
	    for(int i=1;i<N;i++)
	    {
	        if(a[i]>max)
	        max=a[i];
	    }
	    int min=a[0];
	    for(int i=1;i<N;i++)
	    {
	        if(a[i]<min)
	        min=a[i];
	    }
	    cout<<max<<" ";
	   cout<<min<<endl;
	}
	return 0;
}
